docs/integrations/destinations/deepset.md

```diff
@@ -0,0 +1,40 @@
+# deepset
+
+## Prerequisites
+
+- 
+
+## Sync overview
+
+### Output schema
+
+- 
+
+### Features
+
+| Feature                        | Support | Notes                                                                                   |
+| :----------------------------- | :-----: | :-------------------------------------------------------------------------------------- |
+| Full Refresh Sync              |   ✅    | Warning: this mode deletes all previously synced data in the configured workspace.      |
+| Incremental - Append Sync      |   ✅    |                                                                                         |
+| Incremental - Append + Deduped |   ❌    |                                                                                         |
```

Comment:
We've implemented append + dedupe for Motherduck destination very recently — if you're interested in implementing it, there are some examples there. Docs are pretty outdated.